File: KOTLIN/1009.kt
import java.util.Scanner

fun main(args:Array<String>){
  val sc = Scanner(System.`in`)  

  val name :String = sc.next()
  var salary : Double = sc.next().toDouble()
  var sales : Double = sc.next().toDouble()
  
  println(saida(salaryPlusBonus(salary, sales)))
  
}

fun saida(vlr :Double ): String = "TOTAL = R$ %.2f".format(vlr)
fun salaryPlusBonus(salary:Double, sales:Double):Double =  salary + sales*.15

File: GO/1009.go
package main

import "fmt"

func Salary(base, bonus float64) float64 {
	return base + bonus*0.15
}
func main() {
	var name string
	var base, bonus float64

	fmt.Scan(&name)
	fmt.Scan(&base)
	fmt.Scan(&bonus)

	fmt.Printf("TOTAL = R$ %.2f\n", Salary(base, bonus))
}
